README.md

```diff
@@ -59,11 +59,11 @@ style blend weight of 0.2:
 
 ## Requirements
 
-* TensorFlow
-* SciPy
-* Pillow
-* NumPy
-* [Pre-trained VGG network][net] (MD5 `8ee3263992981a1d26e73b3ca028a123`)
+* [TensorFlow](https://www.tensorflow.org/versions/r0.11/get_started/os_setup.html#download-and-setup)
```

Comment:
Could we link to https://www.tensorflow.org/versions/master/get_started/os_setup.html#download-and-setup instead?